src/cli/useInteractionReducer.ts

```diff
@@ -69,7 +72,10 @@ function backendCommitGraphToDisplayCommits(
     depth: number,
     hasFork: boolean,
   ): DisplayCommit[] {
-    const sortedChildren = subgraphBackendRootCommit.childCommits.sort((a, b) =>
+    const childCommits = subgraphBackendRootCommit.childCommits.map(
```

Comment:
So we are no longer using ChildCommits in a nested graph-y way, but just as an array of hash commits?

Reply:
Yeah commits will only store their children commit hashes, and we’ll have a big object on Repository that contains all the commits